ios/browser/web/web_state/brave_web_state.h

```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#import <Foundation/Foundation.h>
+
+#ifndef BRAVE_IOS_BROWSER_WEB_WEB_STATE_BRAVE_WEB_STATE_H_
+#define BRAVE_IOS_BROWSER_WEB_WEB_STATE_BRAVE_WEB_STATE_H_
+
+NS_ASSUME_NONNULL_BEGIN
+
+NS_SWIFT_NAME(BraveWebState)
+OBJC_EXPORT
+@interface BraveWebState : NSObject
```

Comment:
Lets avoid prefixing names of stuff that aren't Brave specific features with `Brave`, this is a wrapper around Chromium web state right?

Reply:
Well, I didn't want any confusion with `web::WebState` if someone did `using namespace web;` or `using web::WebState` locally in a function.
But fixed anyway.